packages/sanity/src/core/releases/util/getReleaseIdFromName.ts

```diff
@@ -0,0 +1,5 @@
+import {RELEASE_DOCUMENTS_PATH} from '../store/constants'
+
+export function getReleaseIdFromName(name: string) {
```

Comment:
😭 god I hate these shenanigans
I _think_ that keeping the name as is is fine.
The fact that is a document can be added as context in a comment related to the function (imo)

Not married to this opinion though

Reply:
I'll bring this to slack, we need to be all aligned or it will make us waste a lot of time https://github.com/sanity-io/sanity/pull/7936#discussion_r1867905756